src/exchangeGroups.js

```diff
@@ -91,9 +92,7 @@ export function rabbotConfigFromExchangeGroups(exchangeGroups) {
     if (g.retryExchange) {
       const retryExchange = Object.assign({}, exchangeDefaults, g.retryExchange);
       const retryQueue = Object.assign({}, queueDefaults, g.retryQueue);
-      retryQueue.messageTtl = g.retryDelay;
```

Comment:
don't you need to keep this so that existing retry queues don't mismatch?

Reply:
Oh - I see, it's managed up above, right?

Reply:
Yes

Reply:
Difference is now it's controlled by normalized.retries instead of g.retryExchange. I guess those are the same object?

Reply:
From same config property. Now it's decided at the time of normalization

Reply:
g.retryQueue and normalized.retryQueue are same object